ray-curator/ray_curator/stages/filters/__init__.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .classifier_filter import FastTextLangId, FastTextQualityFilter
+from .code import (
```

Comment:
Also what do we think of really having all text filters be accessible from this modules namespace? What if the user imports it `from nemo_curator.filters.text.code import X; from nemo_curator.filters.heuristic_filter import Y; from nemo_curator.filters.text.fasttext import Z`. Will be nice separation of concerns IMO

Reply:
My main opposition here is that the import paths are long, and that all of these are `DocumentFilter` so do not need to be separated semantically IMO. It may feel a bit too arbitrary.